fe/src/test/java/org/apache/doris/analysis/AggregateTest.java

```diff
@@ -0,0 +1,102 @@
+// Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
I think we can just move this test to `QueryPlanTest.java`

Reply:
I think this is an exception in the analysis phase, not execution plan phase. 
This file is specially used for unit tests related to aggregate function analysis